### ISSUE TITLE
Support video resizing before computing flow

### ISSUE THREAD

**Issue by willprice** [willprice/flowty#39]
When computing flow it is useful to be able to resize the video frame before input into the algorithm

**Comment by sh0gg0th** [comment]
This would be super helpful to have, thanks in advance

**Comment by willprice** [comment]
Hi @sh0gg0th ,
Just a heads up I probably won't work on this for quite a while. Also, it's worth noting that if you want to use flowty for TVL1 it's very slow. OpenCV has a serious performance regression in v4 (bug logged [here](https://github.com/opencv/opencv_contrib/issues/2459)).
I have an alternate optical flow tool (with a not so nice UX) [here](https://github.com/dl-container-registry/furnari-flow) that uses OpenCV 2 which is substantially faster for TVL1.

**Comment by sh0gg0th** [comment]
Hi @willprice - thanks a lot! Yeah I noticed the same problem with speed and was wondering if it had something to do with my CUDA/Docker/drivers setup. Does your furnari-flow build support the latest architectures such as Turing (7.5)?

**Comment by willprice** [comment]
I don't believe it's an issue with yours (or my) setup, but a performance regression with OpenCV itself sadly :( My CUDA chops aren't good enough to debug it sadly.
The docker images for furnari-flow have binaries up to compute 6.1 or so, so not the newest, however there are PTXs inside too so it'll JIT compile for your architecture on the first run (which will take an age) but then these are cached in further invocations (I've got the nvidia PTX cache set up as a docker volume for this purpose--check the dockerfile for details).

**Comment by sh0gg0th** [comment]
Understood, thank you. Could you please point me to your dockerfile for willprice/opencv2-cuda8 ?

**Comment by willprice** [comment]
Sure, here it is: https://github.com/dl-container-registry/opencv2

**Comment by sh0gg0th** [comment]
Excellent, and thanks for all your work!

**Comment by willprice** [comment]
No problem :) let me know if you have any problems
